src/blocks/author-list/components/DisplayStyleSelector.js

```
@@ -8,48 +8,50 @@ import { columns, tablet } from '@wordpress/icons';
 /**
  * Display style selector component for the Author List block.
  *
- * @param value
- * @param onChange
- * @param {Object} props Component props.
+ * @param          value.value
```

ESLint

[error] Missing JSDoc @param "value.value" type.

(jsdoc/require-param-type)